src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnector.java

```diff
@@ -72,9 +73,23 @@ public void start(Map<String, String> props) {
         this.props = props;
         LOGGER.debug("Props " + props);
         Configuration config = Configuration.from(this.props);
+
+        String streamId = config.getString(YugabyteDBConnectorConfig.STREAM_ID);
+        String tableIncludeList = config.getString(YugabyteDBConnectorConfig.TABLE_INCLUDE_LIST);
+        
+        // For CQL tables as well as for the config not using Publication/Replication streamId will be non null
+        usePublication = streamId == null || streamId.isEmpty();
```

Comment:
I would prefer that `usePublication` be a configuration property rather than us evaluating based on this condition. It would have 2 direct impact:
1. The user will be knowing for sure that if they want to use publication then they can provide the config and skip stream ID. We can also add validation in the configuration class that is this `use.publication` or whatever name is given, if it is set to `true` then stream ID shouldn't be set and we can throw out the error early. I am saying this to keep it till we completely transition to the slot model.
2. Once that is done, we can simply use a getter method i.e. `config.shouldUsePublication()` everywhere we pass the variable `usePublication`

Reply:
We wanted to make the connector configs to be exactly similar to the debezium postgres connector configs. Adding a new config property `usePublication` will create a difference between the two.